docs/endpoints/communities/communities.md

```diff
@@ -2,6 +2,10 @@
 
 ## GET
 
+::: info
+This endpoint provides a summary view of Community objects, ideal for listing multiple communities such as on a homepage or in search results. Detailed attributes like moderators, rules, and report details may be omitted in this summary view.
+:::
+
 Returns an array of Community objects.
```

Comment:
@noClaps, visually wise would you prefer this below or above the info container

Reply:
I'd prefer to have the text above and the container below.